samples/policy.ts

```diff
@@ -0,0 +1,60 @@
+import * as common from './common';
+import * as nodeApi from 'azure-devops-node-api';
+
+import * as PolicyApi from 'azure-devops-node-api/PolicyApi';
+import * as PolicyInterfaces from 'azure-devops-node-api/interfaces/PolicyInterfaces';
+import * as VSSInterfaces from 'azure-devops-node-api/interfaces/common/VSSInterfaces';
+
+export async function run(projectId: string) {
+    const webApi: nodeApi.WebApi = await common.getWebApi();
+    const policyApiObject: PolicyApi.IPolicyApi = await webApi.getPolicyApi();
+
+    common.banner('Policy Samples');
+
+    common.heading('Create Policy Configuration for this Project');
+    const newConfiguration: PolicyInterfaces.PolicyConfiguration = {isEnabled: true,
+                                                                    isBlocking: false,
+                                                                    type: {
+                                                                        id: "fa4e907d-c16b-4a4c-9dfa-4906e5d171dd",
+                                                                        displayName: undefined,
+                                                                        url: undefined
+                                                                      },
+                                                                    settings: {
+                                                                        minimumApproverCount: 1,
+                                                                        creatorVoteCounts: false,
+                                                                        scope: [{
+                                                                            repositoryId: null,
+                                                                            refName: "refs/heads/master",
+                                                                            matchKind: "exact"
+                                                                        }]},
+                                                                    _links: undefined,
+                                                                    createdBy: undefined,
+                                                                    createdDate: undefined,
+                                                                    isDeleted: undefined,
+                                                                    revision: undefined,
+                                                                    id: undefined,
+                                                                    url: undefined
+                                                                    };
+    await policyApiObject.createPolicyConfiguration(newConfiguration, projectId);
+
+    common.heading("Get Policy Configurations for this Project");
+    const configurations: PolicyInterfaces.PolicyConfiguration[] = await policyApiObject.getPolicyConfigurations(projectId);
```

Comment:
I wrote it assuming we are, I guess it doesn't really need us to though. My only concern with doing this on a project that isn't the one we just created is if it gets stopped in the middle it would leave this policy on their project. Probably not a huge deal, but I still like the idea of spinning up our own.

Reply:
Definitely agree, just wanted to be sure. I think the way you're doing it is the right way.